modules/op-configuring-basic-authentication-for-git.adoc

```diff
@@ -0,0 +1,84 @@
+// This module is included in the following assembly:
+//
+// *openshift-docs/cicd/pipelines/authenticating-pipelines-using-git-secret.adoc
+
+[id="op-configuring-basic-authentication-for-git_{context}"]
+= Configuring basic authentication for Git
+
+For basic authentication, {pipelines-shortname} creates a `~/.gitconfig` file containing the credentials specified in the secret, before executing any steps in a task run or a pipeline run. When the steps execute, {pipelines-shortname} uses those credentials to retrieve the pipeline resources specified in a run.
+
+[discrete]
+.Procedure
+
+To configure a Git secret of the `basic-auth` type:
+
+. In the `secret.yaml` file, specify the username and password required to access the target Git repository.
```

Comment:
Do not use a lead-in sentence in the Prerequisites or Procedure sections of a module unless it is necessary to aid navigation or add clarity.
https://redhat-documentation.github.io/supplementary-style-guide/#lead-in-sentences